lib/utils.js

```diff
@@ -233,6 +233,10 @@ exports.installHooks = function (hooks, root) {
         var hook = hooks[i];
         var dest = Path.join(hookRoot, hook);
 
+        if (!exports.isDir(hookRoot)) {
+            internals.mkdir(hookRoot);
+        }
+
```

Comment:
This should go before the for loop executes, as it is now it'll run this check on every iteration